MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -69,6 +86,22 @@ - (BOOL)setAutomaticVideoChange:(BOOL)automaticVideoChange {
     return _automaticVideoChange;
 }
 
+- (NSString *)softwareName {
```

Comment:
Manually implemented in for static lib compatibility (though I've not checked if this is still a restriction with the newer compiler)